docs/source/EccFramework.rst

```diff
@@ -0,0 +1,201 @@
+ECC Framework: Automatic Implementation and Evaluation of Error-Correcting Codes for Quantum Computing
+======================================================================================================
+
+The QECC library offers means for automatic implementation and evaluation of error-correcting codes for quantum computing. More precisely, the library allows to automatically apply different error correction schemes to quantum circuits provided as openqasm files. The "protected" quantum circuits can then be exported again in the form of openqasm files or can be directly used for noise-aware quantum circuit simulation. For the latter case, we also provide a wrapper script which makes use of the provided framework to apply error correction schemes to circuits and directly simulate those circuits using qiskit.
+
+Installation
+############
+
+Installation for Users
+----------
+
+.. code-block:: console
+
+  (venv) $ pip install mqt.qecc
+
+
+Installation for Developers
+----------
+
+If you have not done so already, clone the repository using:
+
+.. code-block:: console
+
+  git clone --recurse-submodules https://github.com/cda-tum/qecc
+
+Make sure you are in the main project directory for the next steps.
+
+.. code-block:: console
+
+  cd qecc
+
+and (if necessary), update the submodules.
+
+.. code-block:: console
+
+  git submodule update --init --recursive
+
+Then, the ECC framework can be installed using pip
+
+.. code-block:: console
+
+  (venv) pip install --editable .
+
+If you want to use Qiskit for quantum circuit simulation, you need to install it as well
+
+.. code-block:: console
+
+  (venv) pip install qiskit
+
```

Comment:
This is completely redundant as it is already contained in the QECC documentation.

Reply:
I removed this part of the documentation.